css/store.css

```diff
@@ -0,0 +1,238 @@
+/* store page */
+
+/* main */
+.storemain {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+}
+
+/* section one */
+.formsection {
+  width: 50%;
+  display: flex;
+  flex-direction: column;
+  padding: 2rem;
+  gap: 30px;
+}
+/* form */
+form {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+}
+
+/* form title */
+.formsection h1 {
+  color: var(--orange);
+}
+
+/* form classes */
+label {
+  font-size: 12px;
+  font-weight: 400;
```

Comment:
No need to set this, as the font-weight is already regular by default, and this also sets it to regular